paper.md

```diff
@@ -0,0 +1,113 @@
+
+### 7. 
+
+![View metadata for each image using "image_hash.json" (using https://jsoneditoronline.org/).\label{fig:example7}](https://user-images.githubusercontent.com/26821843/188520860-629c529d-dc5e-4e93-9beb-b65e4560bc13.png)
+
+### 8. 
+
+![View metadata for all images in "summary.json" (using https://jsoneditoronline.org/).\label{fig:example8}](https://user-images.githubusercontent.com/26821843/188520906-3061ecce-cb0e-4c76-9b81-303731110380.png)
+
+# Method
+
+## Workflow Overview
+Trash AI is trained on the [TACO dataset](http://tacodataset.org/) using [YOLO 5](pytorch.org). Trash AI stores images in [IndexDB](https://developer.mozilla.org/en-US/docs/Web/API/IndexedDB_API) to keep the data primarily browser side and uses [tensorflow.js](https://www.tensorflow.org/js) to keep analysis browser side too. When images are uploaded to the browser, Trash AI provides the prediction of the model as a graphical output. The raw data from the model and labeled images can be downloaded in a batch download to expedite analyses. Any data uploaded to the platform may be automatically saved to an [S3 bucket](https://aws.amazon.com/s3/) which we can use to improve the model over time.
+
+## AI Training
+The AI model was developed starting with the TACO dataset which was available with a complimentary Jupyter Notebook on [Kaggle](https://www.kaggle.com/datasets/kneroma/tacotrashdataset). An example notebook was referenced which used the default `YOLO v5 model` [@Jocher:2020] as the basic model to begin transfer learning. Next, transfer learning was completed using the entire TACO dataset to import the image classes and annotations in the YOLO v5 model.
+
+## Limitations
+From our experience, the accuracy of the model varies depending on the quality of the images and their context/background. Trash is a word people use for an object that lacks purpose, and the purpose of an object is often not obvious in an image. Trash is a nuanced classification because the same object in different settings will not be considered trash (e.g. a drink bottle on someone's desk vs in the forest laying on the ground). This is the main challenge with any image-based trash detection algorithm. Not everything that LOOKS like trash IS trash. This and other complexities to trash classification make a general trash AI a challenging (yet worthwhile) long term endeavor. The algorithm is primarily trained on single pieces of trash in the image with the trash laying on the ground and the model will likely excel for that use case currently. Additionally, user feedback has shows that the distance of trash from the camera is a critical aspect. The model performs ideally with single pieces of trash in an image less than 1 m away. The model performs less accurately on images when trash which is farther away such as when taken from a vehicle. This is likely due to the training data, TACO dataset, which consists primarily of images of trash close to the camera.
```

Comment:
Sentence 3--suggest comma after 'e.g.':  "...not be considered trash (e.g., a drink bottle on someone's desk vs in the forest..."

Sentence 6--suggest hyphen for 'long term':  "...make a general trash AI a challenging (yet worthwhile) long-term endeavor."

Sentence 7--suggest rewriting as follows:  "The algorithm is primarily trained on single pieces of trash in the image, with the trash laying on the ground; thus, model class prediction of trash will generally be better for  this type of image, than, say, aerial imagery."  (Note--I wouldn't say that the current model 'excels' at correctly classifying trash in this or any other use case, but rather, the accuracy of its classifications will be better for these kinds of images than other types of images containing trash.)

Reply:
Edits made in https://github.com/code4sac/trash-ai/commit/0b4743ed252ad7f7351950aecef4c1c77e8e7aa0. Split up your recommendation for sentence 7 a little so that it wouldn't be as long.